src/components/icon/assets/alarm_c.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`fill="none"` means it has hardcoded fill color. As intended?

Reply:
You are right! It was the way silly figma exportes it. Removed!